README.md

```diff
@@ -266,31 +266,29 @@ Launch deployment, database server IP is required, rest of settings must match y
 
 _**Note:**_ The first deployment could take several minutes as OpenShift is downloading the necessary images.
 
-### Confirm the MIQ pod is bound to the correct SCC
+### Confirm the deployed MIQ pods are bound to the correct SCC
 
-List and obtain the name of the miq-app pod
+Describe all pods and search for Security Policy
 
 ```bash
-$ oc get pod
-NAME                 READY     STATUS    RESTARTS   AGE
-manageiq-0           1/1       Running   0          2h
-memcached-1-mzeer    1/1       Running   0          3h
-postgresql-1-dufgp   1/1       Running   0          3h
+$ oc describe pods | grep -B2 "Security Policy"
+Name:			httpd-1-5pnrz
+Namespace:		miq-dev
```

Comment:
Should we omit namespace here as it can vary between deployments?

Reply:
Certainly, will omit from output